src/utilities/timeline.test.ts

```diff
@@ -1185,3 +1186,55 @@ test('getUniqueColorSchemeForXRangeLayer', () => {
   const existingScheme = (row2.layers[0] as XRangeLayer).colorScheme;
   expect(getUniqueColorSchemeForXRangeLayer(row2)).not.toBe(existingScheme);
 });
+
+describe('getTimeRangeAroundTime', () => {
+  //utility to convert ISO date to ms
+  function getTime(d: string): number {
+    return new Date(d).getTime();
+  }
+  const clipperLaunch = getTime(`2024-10-14T16:06:00Z`);
```

Comment:
Generally we try to avoid mission specific naming in this repo - mind renaming this to a generic time like "activityTime" and removing Clipper references?

Reply:
@AaronPlave aww that was just for fun for a unit test. Ok i can generalize.